### ISSUE TITLE
Suppress console logs

### ISSUE THREAD

**Issue by ErikBean** [mradionov/karma-jasmine-diff-reporter#3]
I would like to be able to use this and suppress output from all console.log statements. I use a few other reporters in conjunction with this one and I am seeing all my log outputs twice in the console

**Comment by mradionov** [comment]
Could you please tell me what reporters do you use? And what is the value of `reporters` option in karma config? Thanks.

**Comment by ErikBean** [comment]
Sure, the reporters we use are [coverage](https://www.npmjs.com/package/karma-coverage), [junit](https://www.npmjs.com/package/karma-junit-reporter), [jasmine-diff](https://www.npmjs.com/package/karma-jasmine-diff-reporter), and [nested](https://www.npmjs.com/package/karma-nested-reporter)

Here is our current karma.conf: 
```javascript
module.exports = function (config) {
   var files = require("./test-dependencies");

    config.set({
        basePath: '../../',

        files: files.jquery.concat(files.bower.js.concat(files.libraries, files.scripts, files.html, files.specs, files.helpers)),

        autoWatch: false,

        frameworks: ['jasmine'],

        browsers: ['PhantomJS'],

        plugins: [
            'karma-phantomjs-launcher',
            'karma-chrome-launcher',
            'karma-jasmine',
            'karma-coverage',
            'karma-junit-reporter',
            'karma-jasmine-diff-reporter',
            'karma-nested-reporter'
        ],

        reporters: ['coverage', 'junit', 'jasmine-diff', 'nested'],

        preprocessors: {
            'yeti/js/**/*.js': 'coverage'
        },

        reportSlowerThan: 100,

        coverageReporter: {
            type: 'lcov',
            dir: 'yeti/specs/reports/'
        },

        junitReporter: {
            outputFile: 'yeti/specs/reports/TEST-results.xml'
        },

        jasmineDiffReporter: {
            matchers: {
                toEqualObject: {
                    pattern: /Expected \n([\s\S]*)\n to equal \n([\s\S]*)\n/,
                    reverse: true
                }
            }
        }
    });
};
```

**Comment by mradionov** [comment]
I tried pretty much the same configuration as yours with my tests, but was unable to reproduce the duplication. The order of reporters in your `reporters` option is just fine, so it should not cause a double log. Can I ask you to try to comment and/or temporary remove some parts of your *karma.conf.js* and run the tests after that, so we could find what exactly is the reason of the issue in your setup? Likely it's a conflict with another reporter, which may exist, because of the `karma-jasmine-diff-reporter` design, but it might be anything else (maybe custom matcher?). Your help will be much appreciated.

And just to clarify: what exactly is displayed twice? Jasmine matcher error message?

**Comment by ErikBean** [comment]
No, the error message just gets displayed once. What gets displayed twice is any console.log statements I have in tests or code under test, e.g. console.log('this is a debug message'); would print twice. I think the problem is with karma-nested-reporter. It prints console.logs in blue, whereas diff-reporter prints them in white. I will try commenting out some stuff in karma.conf and let you know where that gets me.

**Comment by mradionov** [comment]
Thanks a lot, I was able to reproduce the bug, console.log shows up twice for me too.

**Comment by mradionov** [comment]
I've published a fix on npm in version 0.2.1.

**Comment by ErikBean** [comment]
Awesome, works like a charm. Thank you. 
I didn't realize this was a separate issue, but the log lines for the 'reportSlowerThan' option, e.g. `PhantomJS 1.9.8 (Mac OS X) SLOW 0.147 secs: nameOfSlowSpec` are also duplicated. Looks like the nested reporter prints them in yellow and the diff reporter prints them in white. 
Would you like me to open a separate issue for this?

**Comment by mradionov** [comment]
I think it's all kinda related, so I'll reopen this issue. It seems that I'll have to check all possible logging methods to prevent any duplicates, a bit hastened with the fix. Thanks for the note.

**Comment by mradionov** [comment]
Fixed in 0.2.2. *karma-jasmine-diff-print* now should not print anything in the terminal by itself when it is used in conjunction with other reporters.

**Comment by ErikBean** [comment]
Awesome, can confirm this works now, no duplicate console messages. Thank you.
